src/EnergyPlus/HeatBalanceInternalHeatGains.cc

```diff
@@ -76,6 +76,12 @@ void SetupZoneInternalGain(EnergyPlusData &state,
         if (state.dataHeatBal->Zone(ZoneNum).numSpaces > 1) {
             gainFrac = state.dataHeatBal->space(spaceNum).FloorArea / state.dataHeatBal->Zone(ZoneNum).FloorArea;
         }
+        if (std::find(AdjustTankLossMultipliers.begin(), AdjustTankLossMultipliers.end(), IntGainCompType) != AdjustTankLossMultipliers.end()) {
```

Comment:
The method here is sound, but a comment line to explain would be helpful here.
But should this be done at the space level instead of here?  At this point, only zone-level internal gain types call `SetupSpaceInternalGain` directly, so no multiplier adjustment is necessary. However, if at some point in the future the tank loss allows specifying a Zone Name or Space Name, then this adjustment will not get applied.
I think it would be better to move this into `SetupSpaceInternalGain`.